lib/init.coffee

```diff
@@ -1,6 +1,7 @@
-{BufferedProcess, CompositeDisposable} = require 'atom'
+{Directory, BufferedProcess, CompositeDisposable} = require 'atom'
```

Comment:
I know, just was commenting on things I saw that could be changed to improve the package :wink:.